apps/dev/src/combobox/combobox-demo.component.ts

```diff
@@ -26,43 +26,56 @@ import {
   DtCombobox,
   DtComboboxFilterChange,
 } from '@dynatrace/barista-components/experimental/combobox';
+import { DT_OPTION_CONFIG } from '@dynatrace/barista-components/autocomplete';
 
-const allOptions: { name: string; value: string }[] = [
-  { name: 'Value 3', value: '[value: Value 3]' },
+interface ComboboxDemoOption {
+  readonly name: string;
+  readonly value: string;
+}
+
+const allOptions: ComboboxDemoOption[] = [
+  { name: 'Value 1', value: '[value: Value 1]' },
   { name: 'Value 2', value: '[value: Value 2]' },
-  { name: 'Value 31', value: '[value: Value 31]' },
+  { name: 'Value 3', value: '[value: Value 3]' },
   { name: 'Value 4', value: '[value: Value 4]' },
+  { name: 'Value 5', value: '[value: Value 5]' },
+  { name: 'Value 6', value: '[value: Value 6]' },
+  { name: 'Value 7', value: '[value: Value 7]' },
+  { name: 'Value 8', value: '[value: Value 8]' },
+  { name: 'Value 9', value: '[value: Value 9]' },
+  { name: 'Value 10', value: '[value: Value 10]' },
 ];
 
+function optionFilter(filter: string): (o: ComboboxDemoOption) => boolean {
```

Comment:
please provide a small JsDoc that outlines the behaviour of the function (in this case it is a factory as more it is important to describe why) – especially in the dev app or examples it can benefit to understand those function better. 

Please avoid abbrevations as we have this in the [coding guidelines](https://github.com/dynatrace-oss/barista/blob/master/CODING_STANDARDS.md#naming) as it makes hard to understand the code
 ```suggestion
function optionFilter(filter: string): (option: ComboboxDemoOption) => boolean {
```